phpstan-baseline.neon

```diff
@@ -0,0 +1,77 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Parameter \\#4 \\$prev of static method League\\\\OpenAPIValidation\\\\PSR7\\\\Exception\\\\Validation\\\\InvalidCookies\\:\\:becauseValueDoesNotMatchSchema\\(\\) expects League\\\\OpenAPIValidation\\\\Schema\\\\Exception\\\\SchemaMismatch, Throwable\\|null given\\.$#"
+			count: 1
+			path: src/PSR7/Validators/CookiesValidator/ServerRequestCookieValidator.php
+
+		-
+			message: "#^Parameter \\#1 \\$openApiSpec of class League\\\\OpenAPIValidation\\\\PSR7\\\\PathFinder constructor expects cebe\\\\openapi\\\\spec\\\\OpenApi, cebe\\\\openapi\\\\SpecObjectInterface given\\.$#"
+			count: 2
+			path: tests/FromCommunity/Issue79Test.php
+
+		-
+			message: "#^Generator expects value type string, array\\<int, string\\> given\\.$#"
+			count: 7
+			path: tests/FromCommunity/Issue79Test.php
+
+		-
+			message: "#^Parameter \\#1 \\$openApiSpec of class League\\\\OpenAPIValidation\\\\PSR7\\\\PathFinder constructor expects cebe\\\\openapi\\\\spec\\\\OpenApi, cebe\\\\openapi\\\\SpecObjectInterface given\\.$#"
+			count: 4
+			path: tests/PSR7/PathFinderTest.php
+
+		-
+			message: "#^Method League\\\\OpenAPIValidation\\\\Tests\\\\PSR7\\\\Validators\\\\BodyValidator\\\\MultipartValidatorTest\\:\\:dataProviderMultipartRed\\(\\) should return array\\<array\\<string, string\\>\\> but returns array\\<int, array\\<int, string\\>\\>\\.$#"
```

Comment:
i think this kind of problems can be fixed as well

Reply:
Maybe, but I think it doesn't worth the effort.

Reply:
yea, maybe. at least from your side. maybe I'll fix them by myself later